src/ott/neural/methods/flows/genot.py

```diff
@@ -97,6 +97,8 @@ def __init__(
     self.latent_match_fn = latent_match_fn
     self.n_samples_per_src = n_samples_per_src
 
+    if condition_dim is not None:
+      assert condition_dim > 0, "Condition dimension must be positive."
```

Comment:
ah sorry @michalk8 , this `condition_dim` can be 0, but the `self.vf.cond_dim` has to be larger than 0.

Reply:
Fixed, thanks for the comment.